packages/cli/test/sim/multi_fork.test.ts

```diff
@@ -172,9 +174,27 @@ const checkpointSync = await env.createNodePair({
   keysCount: 0,
 });
 
+// TODO: A workaround for this issue for sim tests only
+// 1. Start the execution node and let it connect to network
+// 2. Wait for few seconds
+// 2. And later start the beacon node and connect to network
+// 3. With this delay the execution node would be synced before the beacon node starts
```

Comment:
is it intentional that step 2 is repeated?
```suggestion
// 2. Wait for few seconds
// 3. And later start the beacon node and connect to network
// 4. With this delay the execution node would be synced before the beacon node starts
```